src/daemon/engine/svc/svc_helpers.hpp

```diff
@@ -0,0 +1,38 @@
+//
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+//
+
+#ifndef OCVSMD_DAEMON_ENGINE_SVC_HELPERS_HPP_INCLUDED
+#define OCVSMD_DAEMON_ENGINE_SVC_HELPERS_HPP_INCLUDED
+
+#include "ipc/server_router.hpp"
+
+#include <cetl/pf17/cetlpf.hpp>
+#include <libcyphal/executor.hpp>
+#include <libcyphal/presentation/presentation.hpp>
+
+namespace ocvsmd
+{
+namespace daemon
+{
+namespace engine
+{
+namespace svc
+{
+
+struct ScvContext
```

Comment:
We're missing a lot of documentation. Simple comments on the (single) responsibility of any declared object is generally a minimum bar.

Reply:
ok, agreed